### DISCUSSION TITLE
Issue template forms

### DISCUSSION THREAD

**Discussion by mestiez** [studio-minus/ppg-bugs-and-requests#2448] [Announcements]
Issues can now be filed using template forms, which is a much more user-friendly way to submit issues and makes it harder for very clever people to mess things up.

Hopefully this will put an end to the "please follow the template" hellscape that his issue board has become.

**Comment by ghost** [comment]
![srsly](https://user-images.githubusercontent.com/104321411/202912819-1f746fe5-dad9-42e4-a652-dbf91b3dae64.png)

**Comment by ATPGaming** [comment]
I'm gonna miss when DindinYT37 said "Please use the template." Dindin is awesome, why would you do this to him?

**Reply by TeslaCoil9001** [reply]
Oh god you have Stockholm syndrome for Dindin

**Reply by dinevolence** [reply]
> Oh god you have Stockholm syndrome for Dindin

You don't seem to know what Stockholm syndrome is...

**Reply by dinevolence** [reply]
> I'm gonna miss when DindinYT37 said "Please use the template." Dindin is awesome, why would you do this to him?

It's still going to happen, bet.
